src/pilot/config_idrac.py

```diff
@@ -68,10 +69,16 @@ def parse_arguments():
     return parser.parse_args()
 
 
-def get_pxe_nic_fqdd(fqdd, model_properties, drac_client):
+def get_pxe_nic_fqdd(fqdd, model_properties, ip_service_tag, node_definition):
```

Comment:
What is the reasoning behind changing the method signature here?  It seems much cleaner to leave it as-is and pass drac_client.

Reply:
Upstream drac client does not contain all methods which were used in config_idrac.py.
like 
https://github.com/dsp-jetpack/JetPack/blob/81fff38e72ec60e156934cf1d0363a91e49662ef/src/pilot/config_idrac.py#L173
and 
https://github.com/dsp-jetpack/JetPack/blob/81fff38e72ec60e156934cf1d0363a91e49662ef/src/pilot/config_idrac.py#L176

So the code needs to have 2 drac clients upstream and discover nodes local drac.
For now I have added one upstream, in future when we add all methods in upstream drac then we can remove this code.

Please continue discussion on 
https://github.com/dsp-jetpack/JetPack/pull/202